keyboards/massdrop/ctrl/keymaps/endgame/keymap.c

```diff
@@ -126,6 +110,49 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     */
 };
 
+#ifdef _______
+#undef _______
+#define _______ {0, 0, 0}
+
+const uint8_t PROGMEM ledmap[][DRIVER_LED_TOTAL][3] = {
+    [_FL] = {
+        _______, CORAL,   CORAL,   _______, _______, CORAL,   CORAL,   _______, _______, CORAL,   _______, YELLOW,  YELLOW,           TEAL,    GOLD,    GOLD,
+        _______, _______, PINK,    PINK,    PINK,    _______, _______, _______, _______, GREEN,   GREEN,   GREEN,   GREEN,   _______, TEAL,    TEAL,    TEAL,
+        ORANGE,  ORANGE,  ORANGE,  ORANGE,  ORANGE,  ORANGE,  _______, AZURE,   AZURE,   _______, _______, _______, _______, _______, TEAL,    TEAL,    TEAL,
+        _______, ORANGE,  ORANGE,  ORANGE,  ORANGE,  ORANGE,  _______, _______, _______, _______, _______, _______, _______,
+        _______, ORANGE,  _______, CORAL,   _______, AZURE,   AZURE,   _______, _______, _______, _______, AZURE,                              SPRING,
+        _______, _______, _______,                   _______,                            _______, PINK,    _______, AZURE,            _______, SPRING,  _______
+    },
+    [_ML] = {
+        _______, GOLD,    GOLD,    GOLD,   _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______,
+        _______, MAGENT,  MAGENT,  MAGENT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, MAGENT,  GOLD,    MAGENT, GOLD,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, GOLD,    GOLD,    GOLD,   GOLD,    _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, CORAL,  _______, _______, _______, _______, _______, _______, _______, _______,                            YELLOW,
```

Comment:
Why did you put the color Yellow for Arrow Up and Down since there is not attributed keys on the _ML layer ?

Reply:
That was incomplete, thanks for pointing it out. Meant to have the arrow cluster operate movement like wasd.